README.md

```diff
@@ -62,7 +62,7 @@ There are three basic categories of incomplete functionality:
 The commands ahead assumes that you have Golang installed and a recent
 version of Python 2, `setuptools` and `pip`.
 
-### Method 1: binary package
+### Method 0: binary package
```

Comment:
🤔Are we going to 0-index human docs?

Reply:
I didn't want to change too much to avoid merge conflicts. Just noticed that were were two first methods and didn't have time to proofread all the instructions.